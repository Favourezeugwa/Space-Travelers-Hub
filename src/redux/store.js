import { configureStore } from '@reduxjs/toolkit';
import rocketReducer from './rocket/rocketSlice';

export default configureStore({
  reducer: {
    rocket: rocketReducer,
  },
});
