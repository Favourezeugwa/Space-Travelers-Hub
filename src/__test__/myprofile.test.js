import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MyProfile from '../Components/Profile/myProfile';
import store from '../redux/store';

it('Test MyProfile renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <MyProfile />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
