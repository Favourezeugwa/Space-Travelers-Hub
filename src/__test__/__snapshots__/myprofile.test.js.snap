// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test MyProfile renders correctly 1`] = `
<div
  className="d-flex justify-content-center mt-5 container"
>
  <div
    className="w-25 me-5 list-group"
    onKeyDown={[Function]}
  >
    <h2>
      My Missions
    </h2>
    <p
      className="text-danger"
    >
      Please would you like to join the mission with Favour?
    </p>
  </div>
  <div
    className="w-25 list-group"
    onKeyDown={[Function]}
  >
    <h2>
      My Rockets
    </h2>
    <p
      className="text-danger"
    >
      Hey! Kindly pick a rocket.
    </p>
  </div>
</div>
`;
